CodeGeneration/Sources/generate-swiftsyntax/templates/basicformat/BasicFormatFile.swift

```diff
@@ -110,6 +115,22 @@ let basicFormatFile = SourceFileSyntax(leadingTrivia: generateCopyrightHeader(fo
       """
     )
 
+    DeclSyntax(
+      """
+      open func shouldOmitNewline(_ node: TokenSyntax) -> Bool {
```

Comment:
Can change. But the idea was to have it open, so other people can override it. 
If we change it, I think we should make I public or private

Reply:
Oh, I didn’t realize it was `open`. With it being open, the naming makes sense. I think a little doc comment would help though.

Reply:
What do you think about trying to explain what the method can be used for?

E.g.

```
If this returns true, `BasicFormat` will not wrap `node` to a new line. This can be used to e.g. keep string interpolation segments on a single line.
```